posthog/api/capture.py

```diff
@@ -264,6 +301,12 @@ def get_event(request):
         else:
             events = [data]
 
+        try:
+            events = drop_events_over_quota(token, events, ingestion_context)
+        except Exception as e:
+            # NOTE: Whilst we are testing this code we want to track exceptions but allow the events through if anything goes wrong
+            capture_exception(e)
+
```

Comment:
thought: After we are done with testing, `drop_events_over_quota` should be a generator to save some memory.

Reply:
Quite possibly